week-1-ts-ecosystem/w01e04-light-me-up/src/modules/powerSourceModule.ts

```diff
@@ -0,0 +1,12 @@
+export class PowerSource {
+  private energySupply: number = 100;
+  
+  consume(energy: number) {
+    if (this.energySupply <= 0)
```

Comment:
- jestem zwolennikiem `{}` wszędzie w blokach warunkowych (ale możesz być innego zdania - innej szkoły styleguide)
- jeśli do `if` dopinasz `else` i `else` ma blok - to w mojej opinii `if` też powinien mieć `{}`